cedar-policy/Cargo.toml

```diff
@@ -67,6 +67,14 @@ proptest = "1.0.0"
 name = "cedar_benchmarks"
 harness = false
 
+[[bench]]
```

Comment:
Do we need to add another `[[bench]]` target for every new benchmark? Could become annoying after a while.

Reply:
That's how it works for fuzz targets in cedar-spec